assets/tasks/common.json

```diff
@@ -1910,13 +1914,15 @@
             ],
             "dependsOrder": "sequence",
             "dependsOn": [
+                "template-specific-initial-task",
                 "validate-settings",
                 "validate-arch-amd64",
                 "copy-docker-compose",
                 "pre-cleanup",
                 "build-container-torizon-release-amd64",
                 "push-container-torizon-release-amd64",
                 "pull-container-torizon-release-amd64",
+                "template-specific-final-task",
```

Comment:
We still need then implemented here, like __weak , with nothing only some echo like "No specific tasks implemented for this template" or something.

Then on the project creation we need to check if the template implements it or not, because if we do not do it the templates that do not have them implemented will be tainted with a label that does not exists.

Reply:
Makes sense, will fix it